whole-tale/ngx-dashboard
# Import bdbag + fix log levels

## Problem
Fixes #221 
Fixes #214 

## Approach
* #221: Add a UI function to Run > Files that allows user to import a BDBag-formatted ZIP file.
* #214: Re-order log levels to make more sense (errors should now always be shown, rest are optional)

## How to Test
Prerequisites: at least one Tale created

1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Navigate to Run > Files > External Data for a Tale that you own
4. Expand the dropdown at the top-right
    * You should see a new option to Import BDBag (text TBD)
5. Click Import BDBag and choose a file to upload
    * You should see a notification appear indicating the the BDBag is being uploaded
6. Wait for the upload/import to complete
7. Expand the dropdown at the top-right once more and choose "WT Data Catalog"
    * You should see the Select Data dialog appear
    * You should see the Bag you just imported is now listed here as well
